src/PasswordInput/index.jsx

```diff
@@ -10,231 +10,236 @@
 import React                 from 'react';
 import PropTypes             from 'prop-types';
 
+import { TextInputWithIcon } from '../';
```

Comment:
Although this works, linter is complaining `Dependency cycle detected (import/no-cycle)`, even if we leave `'../index'`. Can you please check if this is ok, and should we do something about this rule?

Reply:
Yeah I did a `yarn upgrade --latest` in the _Nessie-X-alpha0_ branch and it introduced a bunch of weird new rules. Will fix it in the main branch.